src/Migrations/Version20230801110114.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230801110114 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Resumbit Gift Aid claims (MAT-308)';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // SQL copied from \MatchBot\Migrations\Version20230707100241, campaign id changed.
+        $this->addSql(<<<SQL
+            UPDATE Donation
+            SET
+                tbgGiftAidRequestQueuedAt = NULL,
+                tbgGiftAidRequestFailedAt = NULL,
+                tbgGiftAidRequestConfirmedCompleteAt = NULL,
+                tbgGiftAidRequestCorrelationId = NULL,
+                tbgGiftAidResponseDetail = NULL
+            WHERE
+                campaign_id = 4644 AND
+                giftAid = 1 AND
+                tbgShouldProcessGiftAid = 1 AND
+                tbgGiftAidRequestCorrelationId IS NOT NULL AND
+                donationStatus = 'Paid'
+            LIMIT 224;
```

Comment:
@bdsl What about the limit vs. eligible donations on this campaign?

Reply:
Ah right that's copied over wrongly. Looks like there are 97 rows in the DB that match this where clause.
```suggestion
            LIMIT 97;
```